source/_integrations/nextcloud.markdown

```diff
@@ -16,9 +16,9 @@ The `nextcloud` integration pulls summary [Nextcloud](https://nextcloud.com/) in
 
 ## Configuration
 
-This integration requires access to the monitor API of a Nextcloud instance (This is generally an admin user).
+This integration requires access to the monitor API of a Nextcloud instance  Therefore, the account on where you should generate an App password from the Nextcloud web UI, must have `Administrator` privileges. 
```

Comment:
The account doesn't necessarily need to be an admin user for this to work, the account would just need access to the monitor api - which i believe could be a non-admin user with additional monitor permissions. (don't ask me how to give a non-admin user those permissions though because I haven't tried it!)